src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -10,78 +29,97 @@ stop() {
     exit $1
 }
 
-BASE_URL=''
+check_devive_available() {
+    : ${DEVICE:="chrome"}
+    failed=false
+
+    if [ "$DEVICE" == "docker" ]; then
+        [ -x "$(command -v docker)" ] || failed=true
+    else
+        cd $DASH_DIR/frontend
+        npx cypress verify
+
+        case "$DEVICE" in
+            chrome)
+                [ -x "$(command -v google-chrome)" ] || [ -x "$(command -v google-chrome-stable)" ] ] || failed=true
+                ;;
+            chromium)
+                [ -x "$(command -v chromium)" ] || failed=true
+                ;;
+        esac
+    fi
+
+    if [ "$failed" = "true" ]; then
+            echo "ERROR: $DEVICE not found. You need to install $DEVICE or \
+    use a different device. Supported devices: electron (default), chrome, chromium or docker."
+        stop 1
+    fi
+}
+
+CYPRESS_BASE_URL=''
 DEVICE=''
-E2E_LOGIN_USER=''
-E2E_LOGIN_PWD=''
+CYPRESS_LOGIN_PWD=''
+CYPRESS_LOGIN_USER=''
+NO_COLOR=1
 REMOTE='false'
 
+# TODO: remove this
```

Comment:
It's not removed ;)

Reply:
It is still needed since https://github.com/ceph/ceph-build/pull/1552 hasn't been merged yet :)

Reply:
Removed in last push